velox/functions/prestosql/tests/HyperLogLogFunctionsTest.cpp

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "velox/functions/prestosql/HyperLogLogType.h"
+#include "velox/functions/prestosql/hyperloglog/SparseHll.h"
+#include "velox/functions/prestosql/tests/FunctionBaseTest.h"
+#define XXH_INLINE_ALL
+#include "velox/external/xxhash.h"
+
+using facebook::velox::aggregate::hll::DenseHll;
+using facebook::velox::aggregate::hll::SparseHll;
+
+namespace facebook::velox {
+namespace {
+
+template <typename T>
+uint64_t hashOne(T value) {
+  return XXH64(&value, sizeof(value), 0);
+}
+
+class HyperLogLogFunctionsTest : public functions::test::FunctionBaseTest {
+ protected:
+  std::string serialize(int8_t indexBitLength, const SparseHll& sparseHll) {
```

Comment:
Lets make this method const 

`std::string serialize(int8_t indexBitLength, const SparseHll& sparseHll) const`

Reply:
Actually, I'll make it static.